scripts/onboarding/templates/arc-k8s-extension-provisionedcluster-msi-auth/existingClusterOnboarding.json

```diff
@@ -94,8 +94,7 @@
     "clusterResourceGroup": "[split(parameters('clusterResourceId'),'/')[4]]",
     "clusterName": "[split(parameters('clusterResourceId'),'/')[8]]",
     "clusterLocation": "[replace(parameters('clusterRegion'),' ', '')]",
-    "workspaceLocation":"[replace(parameters('workspaceRegion'),' ', '')]",
-    "dcrNameFull": "[Concat('MSCI', '-', variables('workspaceLocation'), '-', variables('clusterName'))]",
+    "dcrNameFull": "[Concat('MSCI', '-', parameters('workspaceRegion'), '-', variables('clusterName'))]",

```

Comment:
similarly, here - The reason for having workspaceLocation parameter to trim the whitespaces. I see you are removing this variable which can break if the workspace region has spaces. Lets keep the change, what we have in ci_prod.